### PR TITLE
fix(wallet-mobile): hardcoded status bar colors on android

### PR DESCRIPTION
**Pull request by SorinC6**
issue: https://emurgo.atlassian.net/browse/YOMO-1228

- for some reason when the background is white on Android the text color is `dark content`  --> the text color is gray. As a temporary fix on Android have hard-coded the status bar colors to always be black background with white text  
- have tried `setTranslucent ` as well but not doing anything- maybe we should add this in Android xml config files 

⚠️ This happens because of the theming part, hardcoded the colors  for now will make the app behave the same as before 

Screenshot 2024-03-06 at 09 21 00

### CODE REVIEW

**Comment by stackchain** [comment]
@SorinC6 can you experiment with expo one?

**Comment by SorinC6** [comment]
> @SorinC6 can you experiment with expo one?

have tried with expo bar but got the same behavior when adding a white background to the status bar, I must be missing something 🤔

File: apps/wallet-mobile/src/components/StatusBar.tsx [hunk 1]
```diff
@@ -13,8 +13,13 @@ export const StatusBar = ({overrideColor}: Props) => {
...
 
   useFocusEffect(() => {
-    if (Platform.OS === 'android') StatusBarRN.setBackgroundColor(overrideColor ?? backgroundColor)
-    StatusBarRN.setBarStyle(isDark ? 'light-content' : 'dark-content')
+    if (Platform.OS === 'android') {
+      StatusBarRN.setBackgroundColor(theme.color['black-static'])
+      StatusBarRN.setBarStyle('light-content')
+    } else {
+      StatusBarRN.setBarStyle(isDark ? 'light-content' : 'dark-content')
+      StatusBarRN.setBackgroundColor(overrideColor ?? backgroundColor)
```
Comment (banklesss) [inline apps/wallet-mobile/src/components/StatusBar.tsx]: `setBackgroundColor` only is applicable to android. A warning is displayed if you use it on iOS

**Comment by stackchain** [comment]
@SorinC6 lint issues.

**Pull request merged**